README.md

```diff
@@ -179,7 +172,7 @@ Or disable all validations for a group of lines:
 
 ## Documentation
 
-Related documentation you may find [here](https://protofire.github.io/solhint/).
+Related documentation you may find [here](https://solhint-commiunity.github.io/solhint-commiunity/).
```

Comment:
Typos

```suggestion
Related documentation you may find [here](https://solhint-community.github.io/solhint-community/).
```

Reply:
thanks. I also removed references to protofire in some html files which are then (probably) used to generate the docs, and updated the license.